data-builder/parse_dinosaur_bone_crafting.py

```diff
@@ -29,24 +32,57 @@ def get_systems_from_page(soup):
 
         body = table.find('tbody')
 
-        system_name = headers[0].getText()
+        system_name = headers[0].getText().rstrip()
 
         systems[system_name] = {'*': []}
 
+        create_slot_specific_systems(system_name, systems)
```

Comment:
In order to facilitate the fact that some dino augments give different benefits depending upon where they are slotted, I added unique slot names for these "alternate" items.